examples/k8s.yaml

```diff
@@ -10,11 +10,6 @@ images:
     arch: "x86_64"
   - location: "https://cloud-images.ubuntu.com/focal/current/focal-server-cloudimg-arm64.img"
     arch: "aarch64"
-mounts:
-  - location: "~"
-    writable: false
-  - location: "/tmp/lima"
-    writable: true
 containerd:
```

Comment:
Please add this note, then LGTM
https://github.com/lima-vm/lima/blob/cf42676298bb29c762ea0cb4803d4c6150147e9c/examples/k3s.yaml#L21-L22